packages/server-side-render/src/index.js

```diff
@@ -44,3 +44,14 @@ export default withSelect(
 		);
 	}
 );
+
+if( window && window.wp && window.wp.components ) {
+	window.wp.components.ServerSideRender = forwardRef( ( props, ref ) => {
+		wp.deprecated( 'wp.components.ServerSideRender', {
```

Comment:
Can you import `deprecated` to ensure it's listed as a dependency of the module when generating deps file for WordPress?

Reply:
PR updated 👍